src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue13258.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if TEST_FAILS_ON_WINDOWS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue13258 : _IssuesUITest
+	{
+		public Issue13258(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "MAUI Slider thumb image is big on android";
+
+		[Test]
+		[Category(UITestCategories.Slider)]
+		public void SliderThumbImageShouldBeScaled()
```

Comment:
Pending snapshot on Android:
![image](https://github.com/user-attachments/assets/cc6ce178-0c44-4db4-99dc-e80dbdcfc1d9)
Could you commit it?

Reply:
The thumb image is clearly blurry. From a big image, related with the resize code?

Reply:
@jsuarezruiz , I have updated the size of the thumb to align with the Material 2 design specifications. I implemented new logic to preserve the quality of the image, as it was previously blurry due to resizing from a large image. please let me know if you have any concerns.